client/src/models/solid/Profile.ts

```diff
@@ -30,19 +30,19 @@ export const create: Create = async (apiType, params) => {
     api.union(extrusion, true, copy)
   }
   api.clearSolid(copy)
-  return extrusion
+  return [await extrusion]
 }
 
-export const getScene = async (solidId: number, api: ApiNoHistory) => {
+export const getScene = async (solidIds: number[], api: ApiNoHistory) => {
   if (!api) return
-  const scene = await api.createScene(solidId)
+  const scene = await api.createScene(solidIds)
   scene && colorize(scene)
   return scene
 }
 
 const colorize = (scene: THREE.Scene) => {
   const customRed = new Color('rgb(203, 67, 188)')
-  setNodesColor('Solid', customRed, scene)
+  setSolidsColor('Solid0', customRed, scene)
```

Comment:
The same as above: https://github.com/awv-informatik/buerli-examples/pull/11/files#r1039857321